docs/user-guide/plugins.md

```diff
@@ -32,3 +32,4 @@ Plugins are rules and sets of rules built by the community that support methodol
 -   [`stylelint-value-no-unknown-custom-properties`](https://github.com/csstools/stylelint-value-no-unknown-custom-properties): Disallow unknown custom properties.
 -   [`stylelint-z-index-value-constraint`](https://github.com/kristerkari/stylelint-z-index-value-constraint): Specify minimum and maximum constraint value for z-index.
 -   [`stylelint-itcss`](https://github.com/KamiKillertO/stylelint-itcss): A set of rules to enforce ITCSS architecture.
+-   [`stylelint-plugin-ecss`](https://github.com/tyankatsu0105/stylelint-plugin-ecss):  Stylelint rule for [`ecss`](http://ecss.io/). (To recommend use with [`stylelint-config-ecss`](https://github.com/tyankatsu0105/stylelint-config-ecss))
```

Comment:
I would suggest the following wording. Also removed unrelated info.

```suggestion
-   [`stylelint-plugin-ecss`](https://github.com/tyankatsu0105/stylelint-plugin-ecss): A set of rules to enforce [`Enduring CSS`](http://ecss.io/) architecture.
```

Reply:
@hudochenkov 
Thanks !! 
I fixed the word :)